packages/app/src/components/Footer/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { AccessibilityNew, BugReport } from "@mui/icons-material";
+import { Box, Button, Typography } from "@mui/material";
+import vdl_logo from "../../assets/vdl_logo.svg";
+import { accessibilityStatementAtom } from "../../atoms/accessibilityStatementAtom";
+import { useRecoilState } from "recoil";
+import { AccessibilityStatement } from "../AccessiblityStatement";
+import { About } from "../About";
+import { aboutAtom } from "../../atoms/aboutAtom";
+
+const Footer = () => {
+
+    const categoryCSS = {
+        display: "flex",
+        justifyContent: "space-around",
+        alignItems: "center",
+    }
+
+    const [ accessibilityStatement, setAccessibilityStatement ] = useRecoilState(accessibilityStatementAtom);
+    const [ aboutModal, setAboutModal ] = useRecoilState(aboutAtom);
+
+    const openAccessibilityStatement = () => {
+        setAccessibilityStatement(true);
+    }
+
+    const closeAccessibilityStatement = () => {
+        setAccessibilityStatement(false);
+    }
+
+    const openAboutModal = () => {
+        setAboutModal(true);
+    }
+
+    const closeAboutModal = () => {
+        setAboutModal(false);
+    }
```

Comment:
Since you probably only use these in 1 place, I'd argue these are unnecessary. For example, I would just use the `setAboutModal()` in the `onclick`s